src/main/java/com/freenow/apis/phrase/tasks/PhraseAppSyncTask.java

```diff
@@ -68,12 +70,27 @@ public PhraseAppSyncTask(final String authToken, final String projectId)
     */
     public PhraseAppSyncTask(final String authToken, final String projectId, final String scheme, final String host)
     {
-        this(authToken, projectId, Arrays.asList(DEFAULT_BRANCH), scheme, host);
+        this(authToken, projectId, null, singletonList(DEFAULT_BRANCH), scheme, host);
     }
 
-    public PhraseAppSyncTask(final String authToken, final String projectId, final List<String> branches, final String scheme, final String host)
+
+    /**
+     * Constructs a new PhraseAppSyncTask which downloads translation from a project tag.
+     * @param authToken Phrase auth token (optional)
+     * @param projectId ID of the project in Phrase
+     * @param tags List of comma separated tag names of the project in Phrase
+     * @param scheme URL scheme of the Phrase API, e.g. {@code https}
+     * @param host Host of the Phrase API, e.g. {@code api.phraseapp.com}
+     */
+    public PhraseAppSyncTask(final String authToken, final String projectId, final String tags, final String scheme, final String host)
```

Comment:
Exposing `scheme` and `host` as two separate parameters has lead to an issue in the past. Consumers of this API had to add custom logic to split a URL into scheme and host. That custom logic then did not account for the possibility of scheme being `http` instead of `https`, leading to lots of extra modifications for over 20 services.

#33 added a new method that accepts a base URL and does the splitting on its own. Would prefer that this method does the same.

Reply:
I understand. I updated the constructors to follow #33.

Also, I removed constructors using `schema` and `host` because there were conflicts having constructors with the same number of parameters, and the same types.

Reply:
> I removed constructors using `schema` and `host`

Too bad that it requires removing the constructors. There are some services in our backend that rely on that API. I'm not sure if they use version 1.x or version 2.x of this library.

The right thing to do, in terms of SemVer, would be to release this as v3.x. What do you think?

Reply:
Agree. Done!